Automattic/wp-calypso
# Podcasting: Update My Home to display launched podcast site message

#### Changes proposed in this Pull Request

* Display "You launched your podcast site!" title and description for "Site created" step when the podcasting site has been launched.

#### Testing instructions



* Create a podcast site, but don't launch it.
* With this branch checked out, go to "My Home" for the site you created (http://calypso.localhost:3000/home/your-podcast-site-slug.wordpress.com)
* Click on the "Site Created" step under Site Setup if a different step is visible. You should see the podcast welcome message:
<img width="1065" alt="Screen Shot 2021-02-12 at 4 22 01 PM" src="https://user-images.githubusercontent.com/1689238/107823848-7b649c80-6d4e-11eb-92ba-ddc8efa82158.png">

* Launch the site (e.g. from the editor or Edit Homepage).
* When you land on "My Home" again, click "Show Site Setup" and then on the "Site Created" step under Site Setup if a different step is visible. You should see the podcast launched message:
<img width="1062" alt="Screen Shot 2021-02-12 at 4 25 33 PM" src="https://user-images.githubusercontent.com/1689238/107824147-f6c64e00-6d4e-11eb-8d02-8c364ca49328.png">




Related to 437-gh-Automattic/dotcom-manage
